app/news/list/news-page.module.css

```
@@ -0,0 +1,90 @@
+  text-decoration: underline;
+}
+.newsList li {
+  display: list-item; /* ← これで li が横に広がらなくなる */
+}
+
+.backButton {
+  position: fixed;
+  bottom: 20px;
+  right: 10%;
+
+  width: 120px;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)